### DISCUSSION TITLE
No response from GitHub Support

### DISCUSSION THREAD

**Discussion by ozlemdemirkiran** [community/community#145770] [Other Feature Feedback, Questions, & Ideas]
Hi everyone,

We are facing an issue with our GitHub account being flagged, which has caused several limitations. Most urgently, we are unable to add new team members to our organization, which is critically impacting our workflow and bringing operations to a standstill.

We submitted a support ticket a week ago but have not received a response yet. Has anyone experienced a similar issue? Are there any alternative ways to escalate this problem or speed up the resolution process?

Your advice or suggestions would be greatly appreciated!

**Comment by mresitgokce1** [comment]
+1

**Comment by ghost** [comment]
+1

**Comment by MatinGhanbari** [comment]
Don't worry these are the christmas days so that is why everything is blocked XD
I hope they will reply your ticket soon, best of luck :)

**Comment by ozlemdemirkiran** [comment]
I got feedback and found the solution. Good luck to everyone.

**Reply by MatinGhanbari** [reply]
Please if you find my answer helpful mark my reply as an answer.
Thank you.

**Comment by AmineRACHID** [comment]
+1
